src/prelude/prim.mo

```diff
@@ -50,6 +50,10 @@ func debugPrintNat(x : Nat) { debugPrint (@text_of_Nat x) };
 func debugPrintInt(x : Int) { debugPrint (@text_of_Int x) };
 func debugPrintChar(x : Char) { debugPrint (charToText x) };
 
+// Trapping
+
+func trap(x : Text) { (prim "trap" : Text -> None) x };
```

Comment:
Shouldn't that be: 
func trap(x : Text) : None { (prim "trap" : Text -> None) x };

Reply:
Yes, sorry, my bad